### ISSUE TITLE
IRA patch fails

### ISSUE THREAD

**Issue by alpyre** [cahirwpz/amigaos-cross-toolchain#97]
On: 
Ubuntu 16.04.4 LTS (xenial)
GNU patch 2.7.5

`./toolchain-m68k --prefix=/opt/m68k-amigaos build`
fails with this error:
```
DEBUG: enter directory ".build-m68k/build"
DEBUG: execute "patch -t -p0 -i /home/alper/Dev/AmigaDev/amigaos-cross-toolchain/patches/ira/Makefile.diff"
can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- ira/Makefile	2015-09-11 09:25:13.000000000 +0200
|+++ ira/Makefile	2015-09-11 09:21:37.000000000 +0200
--------------------------
No file to patch.  Skipping patch.
1 out of 1 hunk ignored
ERROR: command "patch -t -p0 -i /home/alper/Dev/AmigaDev/amigaos-cross-toolchain/patches/ira/Makefile.diff" failed with 1
```

**Comment by alpyre** [comment]
Ok. I've figured out what's going wrong.

On March 6th 2018, Frank Wille updated the ira.lha package on Aminet to a new version which is completely incompatible with the patches in this build.
(However, I highly doubt its compatibility with this toolchain ever let alone the patches).

I could find the earlier version on an Aminet mirror here:
http://ftp.sunet.se/mirror/archive/ftp.sunet.se/pub/os/amiga/aminet/dev/asm/ira.lha
(I suppose it did not sync yet).

I think everything will be fine if you grab and secure this old version and update the download link in the build script.

**Comment by cahirwpz** [comment]
Thanks for the report! Fixed in e8996ad23ea32ad5aaf90364425bf62aabe610ce
